install/windows.md

```diff
@@ -411,7 +411,7 @@ Um den XServer verwenden zu können muss noch eine Einstellung unter Verwendung
 vorgenommen werden.
 Dafür wird Windows Terminal geöffnet und der Befehl
 ```
-echo "export DISPLAY=$(route.exe print | grep 0.0.0.0 | head -1 | awk '{print $4}'):0.0" >> ~/.bashrc
+echo 'export DISPLAY="$(route.exe print | grep 0.0.0.0 | head -1 | awk '{print $4}'):0.0"' >> ~/.bashrc
```

Comment:
The result in the `~/.bashrc` is:
`export DISPLAY="$(route.exe print | grep 0.0.0.0 | head -1 | awk {print }):0.0"`
The `$4` and `' '` in the `awk call` are missing.

Reply:
Using `""` and escaping the `$` in the `awk` call works:
`echo 'export DISPLAY="$(route.exe print | grep 0.0.0.0 | head -1 | awk "{print \$4}"):0.0"' >> ~/.bashrc`

Reply:
Warum muss das erste $ nicht escaped werden???

Reply:
Or even easier:
`echo "export DISPLAY=\$(route.exe print | grep 0.0.0.0 | head -1 | awk '{print \$4}'):0.0" >> ~/.bashrc`

Reply:
dann im string `\'`